app/assets/stylesheets/application.scss

```diff
@@ -649,7 +631,74 @@ details {
   }
 }
 
+.terms-and-definitions {
+  display: block;
+
+  a {
+    padding-right: 0.25em;
+  }
+}
+
+.breadcrumbs__item {
+  max-width: 75%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+// ============================================================================
+// Overridess
+// ============================================================================
+
 .registers-\!-faded-colour {
   color: $grey-1;
   padding-left: 0.25em;
 }
+
+.registers-\!-border-off {
+  border: 0;
+}
+
+// ============================================================================
+// Migration hacks
+// ============================================================================
```

Comment:
will we remove these once (if?) elements is removed?

Reply:
Yes - there's in the next mission for the move to Design System and the removal of Elements.